README.md

```diff
@@ -0,0 +1,70 @@
+# AlamofireGenome
+
+An [Alamofire](https://github.com/Alamofire/Alamofire) extension that automatically converts JSON into Swift objects using [Genome](https://github.com/LoganWright/Genome).
+
+## Installation
+
+### Cocoapods
+
+Add the following to your `Podfile`:
+
+```
+pod 'AlamofireGenome', '~> 1.0'
```

Comment:
What do you think of leaving the version off? I prefer locking to a specific version when there's a reason, but leaving it open if not.